### DISCUSSION TITLE
dxf file -> plt fig save

### DISCUSSION THREAD

**Discussion by wingtgniw** [mozman/ezdxf#1122] [Q&A]
**Describe the bug**
I add texture image by using doc.add_image_def and msp.add_image to dxf file.
And I save image follow this Drawing Add-on tutorial link(https://ezdxf.mozman.at/docs/howto/drawing-addon.html).
But texture image did not be drawn.
I have mistake?

Thanks

**Screenshots**
![image](https://github.com/mozman/ezdxf/assets/53434687/d66f571c-75e4-44bf-90df-0a343ef2c6d1)

**Comment by mozman** [comment]
Images are not supported by all backends, see release notes #1041

**Reply by wingtgniw** [reply]
all right...

So how can I solve bugs related nested hatch?

Like black box which is placed on left middle side.

[walls_polygons.txt](https://github.com/user-attachments/files/16122951/walls_polygons.txt)
this is polygons list file

hatch_style=ezdxf.const.HATCH_STYLE_NESTED

first polygon's flags=ezdxf.const.BOUNDARY_PATH_EXTERNAL and others's flags=ezdxf.const.BOUNDARY_PATH_OUTERMOST

**Reply by mozman** [reply]
Most likely these are artifacts caused by the simple algorithm to detect nested boundaries implemented in `ezdxf` and there is nothing you can do except use a better tool. It could also be due to the backend algorithm that renders the final filled polygons, but also there is nothing you can do about it.

`Ezdxf` is not on par with CAD applications and never will be, this is not a design goal.

**Comment by wingtgniw** [comment]
Thanks for the feedback.

Have a nice day. :)

2024년 7월 8일 (월) 오후 3:11, Manfred Moitzi ***@***.***>님이 작성:
